src/main.rs

```diff
@@ -43,13 +47,24 @@ fn main() -> Result<(), Box<dyn Error>> {
 }
 
 fn print_total(total: usize) {
+    #[cfg(windows)]
+    set_windows_virtual_terminal();
```

Comment:
Sorry, I probably didn’t very clearly describe what I meant.
It seems to me more correct to call the method from the `fn main` body.
Before calling any output functions, so that the order of calling output functions does not affect the output configuration.

Reply:
@DDtKey 
Okay, i get it now. Will change.